3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,113 @@
+//Age-ify (A future age calculator)
+
+const yearOfBirth = 1991;
+const yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+
+console.log(`you will be ${age} years old in ${yearFuture}`);
+
+//Goodboy-Oldboy (A dog age calculator)
+
+const dogYearOfBirth = 2017;
+const dogYearFuture = 2027;
+const shouldShowResultInDogYears = true;
+
+let dogAgeInHumanYears = dogYearFuture - dogYearOfBirth;
+let dogAgeInDogYears = dogAgeInHumanYears * 7;
+
+if (shouldShowResultInDogYears) {
+  console.log(
+    `Your dog will be ${dogAgeInDogYears} dog years old in ${dogYearFuture}`
+  );
+} else {
+  console.log(
+    `Your dog will be ${dogAgeInHumanYears} human years old in ${dogYearFuture}`
+  );
+}
+
+//Housey pricey (A house price estimator)
+
+friendsHouse = [
+  {
+    name: "Peter",
+    housePrice: 2500000,
+    width: 8,
+    depth: 10,
+    height: 10,
+    gardenSizeInM2: 300,
+  },
+  {
+    name: "Julia",
+    housePrice: 1000000,
+    width: 5,
+    depth: 11,
+    height: 8,
+    gardenSizeInM2: 70,
+  },
+];
+
+function calculateHouseCost(person) {
+  return (
+    person.width * person.height * person.depth * 2.5 * 1000 +
+    person.gardenSizeInM2 * 300
+  );
+}
+
+const peterHousecost = calculateHouseCost(friendsHouse[0]);
```

Comment:
This one should be **peterHouseCost**, same for **juliaHouseCost**

Reply:
All changes have been made as per recommendation.